WEB_SERVICE/WEB/_MEDIUM/GRAPHQL_CIRCULAR_FRAGMENTS/meta.json

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "Circular Fragment in GraphQL",
+  "short_description": "Circular fragment references in GraphQL can lead to server-side resource exhaustion and denial of service.",
+  "risk_rating": "medium",
+  "references": {
+    "Circular Fragment Exploitation in GraphQL with BurpSuite": "https://medium.com/@laur.telliskivi/circular-fragment-exploitation-in-graphql-with-burpsuite-42c490ecab98",
+    "Managing Circular Imports in GraphQL Modules": "https://the-guild.dev/blog/manage-circular-imports-hell-in-graphql-modules"
+  },
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "CWE_TOP_25": [
+      "CWE_400"
+    ],
+    "PCI_STANDARDS":[
+      "REQ_6_2",
+      "REQ_6_4",
+      "REQ_11_3"
+    ],
+    "OWASP_MASVS_L2": [
+      "MSTG_PLATFORM_2"
+    ],
+    "OWASP_ASVS_L3": [
```

Comment:
ASVS_L2?

Reply:
An application achieves ASVS Level 3 (or Advanced) if it adequately defends against advanced application security vulnerabilities and also demonstrates principles of good security design. An application at ASVS Level 3 requires more in depth analysis of architecture, coding, and testing than all the other levels.

Reply:
I don't see how is your comment related?

Reply:
I thought you were asking for ASVS_L3.